core/vm/interpreter.go

```diff
@@ -124,6 +124,9 @@ func copyJumpTable(jt *JumpTable) *JumpTable {
 func NewEVMInterpreter(evm VMInterpreter, cfg Config) *EVMInterpreter {
 	var jt *JumpTable
 	switch {
+	// to add our own IsRohan chain rule, we would need to fork or code or chain.Config
```

Comment:
not needed, probably a leftover from the rebase on my refactoring

Reply:
Got it